Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM python:3.12
-FROM integralsw/osa-python:11.1-11-g024d72b4-20200722-185528-refcat-43.0-heasoft-6.28-python-3.8.5
+# using the osa-python-docker image compiled with python3.10 and heasoft 6.32.1
```

Comment:
No I mean that if you keep alternative code commented, it's good to put a comment why.

The kind of comment you put is I think self-evident, so I just propose to drop it.
```suggestion
```

```suggestion
# using the osa-python-docker image compiled with python3.10 and heasoft 6.32.1
```